release_notes/v2_vs_v3.adoc

```diff
@@ -0,0 +1,107 @@
+
+While most other PaaSes limit themselves to web frameworks and rely on external services for other types of components, OpenShift v3 makes even more application topologies possible and manageable.
+
+OpenShift v3 uses the term "application" as a concept that links services together. You can have as many components as you desire, contained and flexibly linked within a link:../architecture/core_concepts/projects_and_users.html#projects[project], and, optionally, labeled to provide grouping or structure. This updated model allows for a standalone MySQL instance, or one shared between JBoss components.
+
+Flexible linking means you can link any two arbitrary components together. As
+long as one component can export environment variables and the second component
+can consume values from those environment variables, and with potential variable
+name transformation, you can link together any two components without having to
+change the images they are based on. So, the best containerized implementation
+of your desired database and web framework can be consumed directly rather than
+you having to fork them both and rework them to be compatible.
+
+The means you can build anything on OpenShift. And that is OpenShift's
+primary aim: to be a container-based platform that lets you build entire
+applications in a repeatable life cycle.
+
+== Cartridges vs Images
+
+In OpenShift v3, an link:../architecture/core_concepts/containers_and_images.html#docker-images[image] has replaced OpenShift v2's concept of a cartridge.
+
+Cartridges in OpenShift v2 were the focal point for building applications. Each cartridge provided the required libraries, source code, build mechanisms, connection logic, and routing logic along with a preconfigured environment to run the components of your applications.
+
+However, cartridges came with disadvantages. With cartridges, there was no clear distinction between the developer content and the cartridge content, and you did not have ownership of the home directory on each gear of your application. Also, cartridges were not the best distribution mechanism for large binaries. While you could use external dependencies from within cartridges, doing so would lose the benefits of encapsulation.
+
+From a packaging perspective, an image performs more tasks than a cartridge, and provides better encapsulation and flexibility. However, cartridges also included logic for building, deploying, and routing, which do not exist in images. In OpenShift v3, these additional needs are met by link:../architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image (S2I)] and link:../architecture/core_concepts/templates.html[template configuration].
```

Comment:
I think "template configuration" is ambiguous.  Suggest "template-based configuration" if configuration is the primary noun, or "configuration of templates" if template is the primary noun.